### ISSUE TITLE
Dragging cell onto a column produces interesting effects

### ISSUE THREAD

**Issue by Ark-kun** [libvips/nip2#116]
Dragging a cell and dropping it on column header did nothing. But dropping it on the input field below inserted some interesting characters: `Xn`
![image](https://github.com/libvips/nip2/assets/1829149/fb8d65e0-1dd3-4fb0-9071-8f64c47e7452)

**Comment by jcupitt** [comment]
Yes, it's a terrible hack. I think cell drag-drop puts a pointer on the clipboard (!!). Hopefully nip4 will fix this.
